exercises/practice/bob/.approaches/switch-on-tuple/content.md

```diff
@@ -0,0 +1,47 @@
+# `switch` on a `tuple`
+
+```csharp
+    // uses System.Linq;
+    private static bool isShout(string input) => input.Any(c => char.IsLetter(c)) && input.ToUpper() == input;
+
+    public static string Response(string statement)
+    {
+        var input = statement.TrimEnd();
+        if (input == "") return "Fine. Be that way!";
+
+        switch ((input.EndsWith('?'), isShout(input)))
```

Comment:
`isShout` was defined separately because I felt it was too long to put inline. I really don't think anyone should have a problem understanding that  `input.EndsWith('?')` is for determining a question, especially since that's how they might implement it. So, creating a small separate method or variable just to use it in one place seems unnecessary to me.

The reason for the `isQuestion` variable in the answer array is because it's a bit more unwieldy to inline as a ternary expression.

Reply:
Okay that's fine with me!